news/_posts/releases/2020-04-17-prestashop-1-7-7-0-beta-release.md

```diff
@@ -0,0 +1,133 @@
+---
+layout: post
+title:  "PrestaShop 1.7.7.0 Beta Is Ready To Be Tested!"
+subtitle: "You really should download it and test it"
+date:   2020-04-17 12:00:00
+authors: [ PrestaShop ]
+image: /assets/images/2020/04/1.7.7_beta_banner.jpg
+icon: icon-leaf
+tags:
+ - 1.7.7.x
+ - version
+ - beta
+ - minor
+ - releases
+---
+
+Go test the first Beta version of PrestaShop 1.7.7.0!
+
+![Prestashop 1.7.7.0 BETA Release](/assets/images/2020/04/1.7.7_beta_banner.jpg)
+
+After more than one year in the making, we are really thrilled to announce that we are releasing the 1.7.7.0 beta. This version is bigger than the 1.7.6.0 with XX merged pull requests, against XX for the 1.7.6.0, and we are still a month away from the final version. 
+
+The community was really engaged with more than 110 external contributors.
+This version includes new features, improvements, bug fixes and cleaner interfaces. We covered essential topics such as order management. By the way, you will enjoy more efficiency in daily tasks with the [full redesign and new features of order pages](). And also more stability with the upgraded compatibility to PHP 7.3 and more flexibility with new hooks.
+ 
```

Comment:
@MatShir Could you add the link to your article when it will be published ?

Reply:
To be improved to give more "sexyness" about why this new version is amazing ;)